website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+See [Launch Templates and Custom Images]({{< ref "/docs/aws/launch-templates/" >}}) for details.
+
+### Can Karpenter deal with workloads for mixed architecture cluster (arm vs. amd)?
+Yes. Build and prepare custom arm images as described in [Launch Templates and Custom Images]({{< ref "/docs/aws/launch-templates/" >}}).
+Specify the desired architecture when you deploy workloads.
+
+### What RBAC access is required?
+!!! NEEDS INFO !!!
+
+### Can I run Karpenter outside of a Kubernetes cluster?
+!!! NEEDS INFO !!!
+
+## Compatibility
+
+### Which versions of Kubernetes does Karpenter support?
+Karpenter is tested with Kubernetes v1.19 and later.
+
+### What Kubernetes distributions are supported?
+Karpenter documents integration with a fresh install of the latest AWS Elastic Kubernetes Service (EKS).
+Existing EKS distributions can be used, but this use case has not yet been documented.
+Other Kubernetes distributions (KOPs, etc.) can be used, but setting up cloud provider permissions for those distributions has not been documented.
+
+### How does Karpenter interact with AWS node group features?
+Provisioners are designed to work alongside static capacity management solutions like EKS Managed Node Groups and EC2 Auto Scaling Groups.
+You can manage all capacity using provisioners, use a mixed model with dynamic and statically managed capacity, or use a fully static approach.
+We expect most users will use a mixed approach in the near term and provisioner-managed in the long term.
+
+
+### How does Karpenter interact with Kubernetes features?
+* Kubernetes Cluster Autoscaler: Karpenter can work alongside cluster autoscaler.
+See [Kubernetes cluster autoscaler]({{< ref "/docs/concepts/#kubernetes-cluster-autoscaler" >}}) for details.
+* Kubernetes Scheduler: Karpenter focuses on scheduling pods that the Kubernetes scheduler has marked as unschedulable.
+See [Scheduling]({{< ref "/docs/concepts/#scheduling" >}}) for details on how Karpenter interacts with the Kubernetes scheduler.
+
+## Provisioning
+### What features does the Karpenter provisioner support?
+See [Provisioner API]({{< ref "/docs/provisioner" >}}) for provisioner examples and descriptions of features.
+
+### Can I create multiple (team-based) provisioners on a cluster?
+Yes, provisioners can identify multiple teams based on labels.
+See [Provisioner API]({{< ref "/docs/provisioner" >}}) for details.
+
+### If multiple provisioners are defined, which will my pod use?
+
+By default, pods will use the rules defined by a provisioner named default.
+This is analogous to the default scheduler.
+To select an alternative provisioner, use the node selector `karpenter.sh/provisioner-name: alternative-provisioner`.
+You must either define a default provisioner or explicitly specify `karpenter.sh/provisioner-name node selector`.
+
+
+### Can I set total limits of CPU and memory for a provisioner?
+!!! NEEDS INFO !!!
```

Comment:
Yes, the setting is provider specific. Here's AWS example https://karpenter.sh/docs/aws/provisioning/#accelerators-gpu